astroquery/mast/observations.py

```diff
@@ -584,7 +584,7 @@ def download_file(self, uri, *, local_path=None, base_url=None, cache=True, clou
 
         return status, msg, url
 
-    def _download_files(self, products, base_dir, *, cache=True, cloud_only=False,):
+    def _download_files(self, products, base_dir, flat=False, *, cache=True, cloud_only=False,):
```

Comment:
Ah, right.  Thanks.  It would be nice to eventually expunge that `*` and be specific about what args are accepted, but I suspect that's a separate issue.

Reply:
I'm not sure I understand what you mean, the * is doing exactly that enforcement of being explicit.

Reply:
I meant that it would be better for documentation (autodoc) to specify all the args that `*` allows, but I guess that varies with dataset as the allowable filters vary?  No matter, not super relevant this PR.  And it's a private method, so not that important for documenting the API.

Reply:
Ah, thank you!  I didn't know about this fun feature.  😄

Reply:
One of my favourites, and would really love to see that we actually get to https://github.com/astropy/astroquery/issues/1746 (mast is already done thanks to Jenny)